Jenkinsfile

```diff
@@ -62,6 +62,13 @@ pipeline {
                 }
             }
             steps {
+                // quick check that no files are modified for the go modules
+                sh 'build/run make -j\$(nproc) mod.check'
+                sh '''#!/bin/bash
+                      echo "IF THIS BUILD STEP FAILS you need to run a clean build and ensure to files are modified. Modified files are printed below."
+                      git status -s | grep M
```

Comment:
It would be safer to grep for `grep '^ M '`

Reply:
If there is any output from `git status -s` it means that we want to fail the build. `grep M` was just a way to get some output and then look at its exit code.